src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -57,15 +57,9 @@ For example, an alert can be triggered when an issue is affecting 10% more uniqu
 
 You can set a trigger for when an issue affects more than {X} percent of [sessions](/product/releases/health/#session) in {time}.
 
-This is approximated by taking the number of sessions in the last hour and scaling it to the alerting window appropriately. For example, if the alert is configured to look at the last 5 minutes, the number of sessions is approximated by taking the number of sessions in the last hour and dividing it by 12.
+_Percent of sessions affected_ is an approximation that's calculated by taking the number of sessions in the last hour and scaling it to the alerting window appropriately. For example, if the alert is configured to look at the last five minutes, the number of sessions is approximated by taking the number of sessions in the last hour and dividing it by 12.
```

Comment:
@mikellykels & @scttcper - can you confirm i haven't changed the meaning here at all? it just read as redundant to me with the note.